cluster.go

```diff
@@ -259,6 +259,10 @@ func (c *clusterClient) _refresh() (err error) {
 			}
 			if len(g.nodes) > 1 {
 				n := len(g.nodes) - 1
+				for i, replica := range g.nodes[1:] {
+					rConn := conns[replica.Addr].conn
+					g.nodes[i+1].AZ = rConn.AZ()
```

Comment:
All the others look good to me. The only thing left is that `AZ()` itself is too costly. Calling `AZ()` sequentially could block the `refresh` for a long time if all the `rConn` haven't actually connected.

I think we should continue to complete the `AZ_AFFINITY` feature in this PR so that we can skip `AZ()` calls when it is not configured on the client. Parallelizing `AZ()` calls can be another PR.

Reply:
[462f2a9](https://github.com/redis/rueidis/pull/721/commits/462f2a920ed06718df23cf0e9ece1fe91201e45c)

I'm perfectly wrong. How about this?